src/sass/modules/_m-footer.scss

```diff
@@ -19,7 +19,94 @@
   }
 }
 
-  .footer-inner {
+.feedback-widget {
+  padding: 1.5em 0;
+  background-color: $color-gray-lightest;
+  color: $color-gray-dark;
+
+  .row {
+    padding: 0 0.9375rem;;
+  }
+
+  .feedback-widget-title {
+    margin: 0 0 0.5em 0;
+  }
+
+  .feedback-button {
+    width: auto;
+    // margin-top: initial;
+  }
+
+  .feedback-widget-form {
+    order: 1;
+    flex: 0 0 100%;
+    // padding-right: 1em;
+  }
+
+  .feedback-widget-need-help {
+    order: 0;
+    flex: 0 1 100%;
+    .feedback-widget-need-help-inner {
+      background-color: $color-aqua-lightest;
+      padding: 1em;
+      margin: 1em 0;
+      h3 {
+        margin-top: 0;
+      }
+    }
+  }
+
+  .feedback-widget-intro {
+    margin: 0;
+  }
+
+  label[for=feedback] {
```

Comment:
It seems odd that you need this much custom css for these fields. The input components we have might help with that.

Reply:
Yeah, I didn't enjoy writing that CSS but I feel like there were a lot of edge cases in that tiny widget with different viewports.